Tests/AppTests/IngestorTests.swift

```diff
@@ -354,12 +364,12 @@ class IngestorTests: AppTestCase {
 
     func test_ingest_unique_owner_name_violation() async throws {
         // Test error behaviour when two packages resolving to the same owner/name are ingested:
-        //   - don't update package
         //   - don't create repository records
```

Comment:
This is an important change to how we're dealing with this error scenario. Previously, we did two things:

- avoid creating a duplicate repository record
- avoid updating the related package

We could preserve this behaviour but it actually simplifies things quite a bit to drop the second part of this. And it's the theme of this change: make sure processing is advanced through a stages despite errors.

In order to ensure this change doesn't break things in the following stage, this test now explicitly runs analysis as part of its validation to ensure it doesn't raise any errors (see below).